### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by iamdevice**
fix validation and run process

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "cron/cron",
+    "name": "iamdevice/cron",
```
Comment (NoUseFreak) [inline composer.json]: I am not accepting this :-)

File: .gitignore [hunk 2]
```diff
@@ -7,3 +7,4 @@
 .vagrant
 Vagrantfile
 /composer.lock
+.idea/
```
Comment (NoUseFreak) [inline .gitignore]: No editor specific stuff in gitignore. Have a look at a global gitignore to ignore your favorite editors. https://help.github.com/articles/ignoring-files/

File: composer.json (hunk 1 above)
Comment (iamdevice) [inline composer.json]: Yeah, your are right ))))
Just i use modified package in other project

Comment (NoUseFreak) [inline composer.json]: This can be done using https://getcomposer.org/doc/05-repositories.md#vcs. Will prevent these mistakes :-)

**Pull request merged**